scripts/release_deb.sh

```diff
@@ -71,9 +71,8 @@ SNAPSHOT=algorand-$(date +%Y%m%d_%H%M%S)
 aptly snapshot create ${SNAPSHOT} from repo algorand
 if [ ! -z "${FIRSTTIME}" ]; then
     echo "first publish"
-    aptly publish snapshot ${SNAPSHOT} "s3:${APTLY_S3_NAME}:"
+    aptly publish snapshot ${SNAPSHOT} "s3:${APTLY_S3_NAME}:" -origin=Algorand -label=Algorand
```

Comment:
aptly needs that to be
`aptly publish snapshot -label="Algorand" -origin="Algorand" ${SNAPSHOT}`

Reply:
Fixed, my mistake.